modules/oadp-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,231 @@
+
+To resolve the issue it is necessary to first restore the whole managed cluster related resources in the following order:
+
+`securitycontextconstraints,customresourcedefinitions,namespaces`,
+
+`managedcluster.cluster.open-cluster-management.io`,
+`managedcluster.clusterview.open-cluster-management.io`,
+`klusterletaddonconfig.agent.open-cluster-management.io`,
+`managedclusteraddon.addon.open-cluster-management.io`,
+
+`storageclasses,volumesnapshotclass.snapshot.storage.k8s.io,volumesnapshotcontents.snapshot.storage.k8s.io,volumesnapshots.snapshot.storage.k8s.io,datauploads.velero.io,persistentvolumes,persistentvolumeclaims,serviceaccounts,secrets,configmaps,limitranges,pods,replicasets.apps,clusterclasses.cluster.x-k8s.io,endpoints,services,-,clusterbootstraps.run.tanzu.vmware.com,clusters.cluster.x-k8s.io,clusterresourcesets.addons.cluster.x-k8s.io`
+
+link:https://issues.redhat.com/browse/OADP-2686[OADP-2686]
+
+
+.Restic restore was partially failing due to Pod Security standard
+
+During interoperability testing, Red Hat OpenShift Container Platform 4.14 had the pod Security mode set to enforce, which caused the pod to be denied. This was caused due to the restore order. The pod was getting created before the Security Context Constraints (SCC) resource, since the pod violated the podSecurity standard it denied the pod. When setting the restore priority field on the Velero server, restore is successful. link:https://issues.redhat.com/browse/OADP-2688[OADP-2688]
+
+.Possible pod volume backup failure if Velero is installed in several namespaces
+
+There was a regresssion in Pod Volume Backup (PVB) functionality when Velero was installed in several namespaces. The pod volume backup controller was not properly limiting itself to PVBs in its own namespace.
+link:https://issues.redhat.com/browse/OADP-2308[OADP-2308]
+
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/browse/OADP-2819?filter=12422837[OADP 1.3.0 resolved issues] in Jira.
+
+[id="known-issues-1.3.0_{context}"]
+== Known issues
+
+For a complete list of all known issues in this release, see the list of link:https://issues.redhat.com/browse/OADP-2688?filter=12422838[OADP 1.3.0 known issues] in Jira.
+
+[id="Upgrade-notes-1.3.0_{context}"]
+== Upgrade notes
+
+[NOTE]
+====
+Always upgrade to next minor version, *do not* skip versions. To update to a later version, upgrade only one channel at a time. For example: to upgrade from OpenShift API for Data Protection (OADP) 1.1 to 1.3, upgrade first to 1.2, then to 1.3.
+====
+
+=== Changes from OADP 1.2 to 1.3
+
+The Velero server has been updated from version 1.11 to https://velero.io/docs/v1.12/upgrade-to-1.12/#upgrade-from-v110-or-higher[1.12]
```

Comment:
remove the link to the upstream upgrade instructions.  Upstream Velero and OADP have VERY different install methods. OADP uses an operator, velero does not.   This can do a lot of harm, please remove and just call out Velero v.1.12

Reply:
@weshayutin 
I'm going to remove the link as well from the 1.2 upgrade notes.